[docs] Update references to expo install/run to npx expo install/run (#22197)
expo/expo: packages/expo-build-properties/README.md

### Installation

> **Note:** To use this config plugin, your apps must be a managed app and build by either [EAS Build](/build/introduction.md) or `npx expo run:[android|ios]`.

Reviewer: ```suggestion
**Note:** To use this config plugin, your apps must be built by either [EAS Build](https://docs.expo.dev/develop/development-builds/introduction/) or `npx expo run:[android|ios]`.
```

Broken link and also I'm not sure if we want to say "managed" in the context of development builds.

Reviewer 2: They are not mutually exclusive at all.

Reviewer: Can you elaborate on that?

Reviewer 2: Adding custom native dependencies that require you to stop using Expo Go and build a development build is unrelated to the workflow. Config plugins _only_ apply to managed workflow apps, and always require a development build.

```

> expo install expo-build-properties

Reviewer: ```suggestion
npx expo install expo-build-properties
```
